contents/docs/integrate/user-properties.mdx

```diff
@@ -5,41 +5,92 @@ sidebar: Docs
 showTitle: true
 ---
 
-In addition to events having properties, users can also have properties. There are three different methods that can be used: `set`, `set_once` and `$unset`. Depending on the integration library the actual function calls look a bit different, but internally they all work the same way. Note that all methods can be used in properties with normal event capture, e.g. using our [js library](/docs/integrate/client/js):
+Users have properties. 
+
+To update the properties of a user you can use: `set`, `set_once` and `$unset`. Depending on the integration library the actual function calls look a bit different, but internally they all work the same way.
+
+- `set`: Set the property even if the property exists on the person
+```js
+posthog.people.set( { location: 'London'  })
+
+// { location: 'London' }
+```
+
+- `set_once`: Update the property if the property does not exist on the person
+```js
+posthog.people.set_once( { initial_location: 'Rome'})
+posthog.people.set_once( { initial_location: 'London'})
+
+// { initial_location: 'Rome' }
+```
+
+// TODO: the unset property doesn't seem to exist?
+- `unset`: Remove a property from a user if exists
+```js
+posthog.people.unset('location')
+```
```

Comment:
@tiina303 and @pjhul this function doesn't seem to exist but I think it should? Or is the only way to set it through the capture function as shown below?